database/index.js

```diff
@@ -11,3 +11,4 @@ dbC.on('error', console.error.bind(console, 'connection error: ')); // eslint-di
 dbC.once('open', () => console.log('connected')); // eslint-disable-line
 
 module.exports.db = db;
+module.exports.dbC = dbC;
```

Comment:
dbC is the variable to connect the database? If so, i figured it out, but if variable name was a it more descriptive it would help - had not seen dbC as convention before, however if dbC is a convention then looks good.

Reply:
I will rename it to dbConnection.